Base/QTGUI/qSlicerExportNodeDialog.cxx

```diff
@@ -465,14 +467,16 @@ const QList<vtkMRMLStorableNode*> & qSlicerExportNodeDialogPrivate::nodeList() c
 //-----------------------------------------------------------------------------
 bool qSlicerExportNodeDialogPrivate::setup(
   vtkMRMLScene* scene,
-  const QList<vtkMRMLStorableNode*> & nodesNonrecursive,
-  const QList<vtkMRMLStorableNode*> & nodesRecursive,
-  vtkMRMLStorableNode* selectedNode /*=nullptr*/
+  const QList<vtkMRMLStorableNode*>& nodesNonrecursive,
+  const QList<vtkMRMLStorableNode*>& nodesRecursive,
+  vtkMRMLStorableNode* selectedNode,
+  const QHash<QString,QVariant>& nodeIdToSubjectHierarchyPath
 )
 {
   this->MRMLScene = scene;
   this->nodesRecursive = nodesRecursive;
   this->nodesNonrecursive = nodesNonrecursive;
+  this->nodeIdToSubjectHierarchyPath = nodeIdToSubjectHierarchyPath;
```

Comment:
I've just noticed that member variables start with lowercase. The coding convention in Slicer is to start member variable names with uppercase. It is not essential, but it would be nice if you could update all the member variable names in this class accordingly.